### PR TITLE
fix: Refactored the app code

### LINKED ISSUE #114: Refactor the app code.
**Describe the chore**
The language/grammar being used in the app needs to be refactored. Almost all sentences currently end with "only". This is one of the issues that need fixing. Refactor any other grammatically wrong strings in the app too.

**Would you like to work on the issue?** (Yes/No)
No

### PR DESCRIPTION
**Pull request by athiranair2000**
Fixed #114 

Changes: Refactored the app code in delete pooja/donation and formatted sentences accordingly.

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -41,27 +41,27 @@
...
+    <string name="update_registered_pooja_details">Update registered pooja details</string>
+    <string name="update_the_details_of_donated_money">Update the details of donated money</string>
     <string name="update">Update</string>
     <string name="error_notSelected">Please select any option</string>
     <string name="fetch_name">name</string>
     <string name="insert_data">Insert Data</string>
     <string name="nothingselected">Nothing selected</string>
     <string name="custompooja">Custom Pooja</string>
-    <string name="amount_in_rs">Amount:                                         Rs.</string>
+    <string name="amount_in_rs">Amount:Rs.</string>
```
Comment (Chromicle) [inline app/src/main/res/values/strings.xml]: revert this change, it has a particular reason for giving some spaces
find it in `ReadSingleData` activity

**Comment by Chromicle** [comment]
And make it semantic commit or PR title

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -3,25 +3,25 @@
     <string name="read">Read</string>
     <string name="read_all">Read All</string>
     <string name="empty">"     "</string>
-    <string name="delete_donated">Selected To Delete Donated Money</string>
-    <string name="delete_pooja">Selected To Delete Registered Pooja</string>
+    <string name="delete_donated">Selected to delete donated money</string>
```
Comment (yashk2000) [inline app/src/main/res/values/strings.xml]: Make this `Delete donated money`

File: app/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -3,25 +3,25 @@
...
+    <string name="delete_pooja">Selected to delete registered Pooja</string>
```
Comment (yashk2000) [inline app/src/main/res/values/strings.xml]: Make this `Delete registered pooja`

File: app/src/main/res/values/strings.xml [hunk 4]
```diff
@@ -3,25 +3,25 @@
...
     <string name="DON">DON</string>
     <string name="wait">Please Wait</string>
     <string name="deleting">Deleting...</string>
     <string name="id_value">IDVALUE</string>
     <string name="json_obj">Json obj</string>
     <string name="result">result</string>
-    <string name="selected_donate">Selected To Donate Money</string>
-    <string name="selected_register">Selected To Register New Pooja</string>
-    <string name="select_pooja_type">Select Pooja Type</string>
+    <string name="selected_donate">Selected to donate money</string>
```
Comment (yashk2000) [inline app/src/main/res/values/strings.xml]: Make this `Donate money`

File: app/src/main/res/values/strings.xml [hunk 5]
```diff
@@ -3,25 +3,25 @@
...
+    <string name="selected_register">Selected to register new pooja</string>
```
Comment (yashk2000) [inline app/src/main/res/values/strings.xml]: Make this `Register new pooja`

File: app/src/main/res/values/strings.xml [hunk 6]
```diff
@@ -3,25 +3,25 @@
...
+    <string name="select_pooja_type">Select pooja type</string>
```
Comment (yashk2000) [inline app/src/main/res/values/strings.xml]: Make this `Pooja type`

File: app/src/main/res/values/strings.xml [hunk 7]
```diff
@@ -3,25 +3,25 @@
...
     <string name="PAID">PAID</string>
     <string name="NOT_PAID">NOT PAID</string>
     <string name="valid_ID">Please enter a valid employee ID</string>
     <string name="insert">Inserting your values..</string>
     <string name="exception">exception</string>
-    <string name="UPDATE">Selected To update donated money details</string>
-    <string name="update_pooja">Selected To update Registered Pooja details</string>
+    <string name="UPDATE">Selected to update donated money details</string>
```
Comment (yashk2000) [inline app/src/main/res/values/strings.xml]: Make this `Update donated money details`

File: app/src/main/res/values/strings.xml [hunk 8]
```diff
@@ -3,25 +3,25 @@
...
+    <string name="update_pooja">Selected to update registered pooja details</string>
```
Comment (yashk2000) [inline app/src/main/res/values/strings.xml]: Make this `Update registered pooja details`

File: app/src/main/res/values/strings.xml [hunk 9]
```diff
@@ -41,27 +41,27 @@
     <string name="records">records</string>
     <string name="no_data">No data found</string>
     <string name="enter_money_donated">Enter Money Donated</string>
-    <string name="people_who_registered_for_pooja">People Who Registered For Pooja</string>
-    <string name="people_who_donated_money_for_temple">People Who Donated Money For Temple</string>
+    <string name="people_who_registered_for_pooja">People who registered for pooja</string>
+    <string name="people_who_donated_money_for_temple">People Who donated money for temple</string>
```
Comment (yashk2000) [inline app/src/main/res/values/strings.xml]: Make the W in `Who` lower case.

File: app/src/main/res/values/strings.xml [hunk 10]
```diff
@@ -41,27 +41,27 @@
...
+    <string name="update_registered_pooja_details">Update registered pooja details</string>
+    <string name="update_the_details_of_donated_money">Update the details of donated money</string>
     <string name="update">Update</string>
     <string name="error_notSelected">Please select any option</string>
     <string name="fetch_name">name</string>
     <string name="insert_data">Insert Data</string>
     <string name="nothingselected">Nothing selected</string>
     <string name="custompooja">Custom Pooja</string>
-    <string name="amount_in_rs">Amount:                                         Rs.</string>
+    <string name="amount_in_rs">Amount:                           Rs.</string>
```
Comment (yashk2000) [inline app/src/main/res/values/strings.xml]: Revert this change entirely. Don't delete any spaces.

Comment (yashk2000) [reply app/src/main/res/values/strings.xml]: `Amount:                                         Rs.`
